tools/packaging/kata-deploy/local-build/kata-deploy-binaries-in-docker.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+#
+# Copyright (c) 2018-2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o errtrace
+
+script_dir=$(dirname "$(readlink -f "$0")")
+kata_dir=$(realpath "${script_dir}/../../../../")
+kata_deploy_create="${script_dir}/kata-deploy-binaries.sh"
+uid=$(id -u ${USER})
+gid=$(id -g ${USER})
+
+TTY_OPT="-i"
+if [ -t 1 ]; then
+	TTY_OPT="-i"
+fi
```

Comment:
You could collapse this down to:

```suggestion
[ -t 1 ] && TTY_OPT="-i"
```